src/components/shared/sidebar/sidebar.tsx

```diff
@@ -66,8 +69,12 @@ function Sidebar(props: any) {
       code === 'dark' ? setTheme('dark') : setTheme('light');
       props.setToggleDis(false);
     }
-    if (title === 'About Map of Pi' && code === 'version') {
-      setShowInfoModel(true); // Show InfoModel popup for App Version
+    if (title === 'About Map of Pi') {
```

Comment:
does it work well with i18n?

Reply:
That will be determined when we start work with internationalizing the app. 🙃